_sass/minimal-mistakes/_page.scss

```diff
@@ -72,6 +72,11 @@ body {
   margin-top: 0;
   line-height: 1;
 
+  a {
+    color: $dark-gray;
```

Comment:
Replace with `$text-color` variable so each of the skins can assign the correct text color to match. They're not all dark gray.